.github/workflows/easy-cp.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+name: Cherry-pick Labeled PR to Release Branch
+
+on:
+  pull_request:
+    branches: master
+    types: [labeled]
+
+permissions: write-all
+
+jobs:
+  cherrypick_to_release:
+    name: cherrypick_to_release
+    runs-on: ubuntu-latest
+    if: |
+      (github.event.label.name == format('cp{0} beta', ':') || github.event.label.name == format('cp{0} stable', ':')) &&
+      (github.event.pull_request.merged == true)
+    steps:
+      - name: Get Release Channel
+        run: |
+          echo "CHANNEL=$(echo ${{ github.event.label.name }} | cut -d ':' -f 2 | xargs)" >> $GITHUB_ENV
+      - name: Get Release Candidate Branch
+        run: |
+          curl -o version.json https://raw.githubusercontent.com/flutter/flutter/$CHANNEL/bin/internal/release-candidate-branch.version
+          echo "RELEASE_BRANCH=$(cat version.json | tr -d '\n')" >> $GITHUB_ENV
+          rm version.json
+      - name: Get Cherry Pick PR
+        run: |
+          echo "COMMIT_SHA=$(echo ${{ github.event.pull_request.merge_commit_sha }})" >> $GITHUB_ENV
+      - name: Checkout Flutter Repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+        with:
+          repository: flutter/flutter
+          token: ${{ github.token }}
+          path: flutter
+          ref: master
+          fetch-depth: 0
+      # use same name when checking out branch, since the marketplace action does a hard reset.
+      - name: Attempt CP
+        id: attempt-cp
+        working-directory: ./flutter
+        run: |
+          git config user.name "GitHub Actions Bot"
+          git config user.email "<>"
+          git fetch origin $RELEASE_BRANCH
+          git fetch origin master
+          git checkout -b cp-$COMMIT_SHA --track origin/$RELEASE_BRANCH
+          git cherry-pick $COMMIT_SHA
+          git status
+      # TODO(xilaizhang): move cp template to flutter/flutter.
```

Comment:
Should this be implemented now?

Reply:
Sure, created https://github.com/flutter/flutter/pull/139590.

The template has to be in markdown format cause the original web form issue template (https://github.com/flutter/flutter/blob/master/.github/ISSUE_TEMPLATE/7_cherry_pick.yml) isn't supported in PR description.

Reply:
yeah I implemented separately cause I thought if the template gets merged into flutter/flutter first, then we can directly use it in github actions. otherwise even if they are in the same PR, the github action still has to download the template from somewhere else because the template doesn't exist on framework yet.